.travis.yml

```diff
@@ -76,6 +78,8 @@ script:
 
     - etc/bin/test-packages
 
+    - gulp admin-css admin-js
```

Comment:
By default it is executed with "watch" task which will stop further processing of travis script. ;)